### PR TITLE
feat: Add audio-text-to-text task.

### PR DESCRIPTION
**Pull request by Vaibhavs10**
There's an increasing number of Audio LMs:

1. https://huggingface.co/collections/reach-vb/ultravox-audio-language-model-release-67373b602af0a52b2a88ae71
2. https://huggingface.co/homebrewltd

And.. many more.

Will Open PRs after this is merged.

### CODE REVIEW

**Review by Vaibhavs10 — COMMENTED** [review]
I mostly based this on `time-series-forecasting` so not sure, if I should update stuff anywhere else.

**Review by Wauplin — COMMENTED** [review]
Looks good to me but better to wait for another approval.

You can also define an icon in this module: https://github.com/huggingface/huggingface.js/blob/main/packages/widgets/src/lib/components/PipelineIcon/PipelineIcon.svelte. For now maybe take the same as `audio-to-audio` or `text-to-speech` ? 
image
image


Also cc-ing @merveenoyan for tasks updates.

File: packages/tasks/src/tasks/index.ts
Comment (merveenoyan) [inline packages/tasks/src/tasks/index.ts:201]: you can put a placeholder here instead

**Review by merveenoyan — APPROVED** [review]
left a small comment, awesome!

File: packages/tasks/src/tasks/index.ts
Comment (Vaibhavs10) [reply packages/tasks/src/tasks/index.ts]: Updated here: https://github.com/huggingface/huggingface.js/pull/1033/commits/77a0ea59400a13d91fe518f1b659a146ec49b46f

File: packages/widgets/src/lib/components/PipelineIcon/PipelineIcon.svelte
Comment (julien-c) [inline packages/widgets/src/lib/components/PipelineIcon/PipelineIcon.svelte:7]: better if the icon is unique imo, but no big deal. cc @gary149

**Review by julien-c — APPROVED** [review]

File: packages/widgets/src/lib/components/PipelineIcon/PipelineIcon.svelte
Comment (Vaibhavs10) [reply packages/widgets/src/lib/components/PipelineIcon/PipelineIcon.svelte]: Fixed: https://github.com/huggingface/huggingface.js/pull/1033/commits/50b60e68e245cbb3072208e8e5ae3e9d037097e2

Comment (Vaibhavs10) [reply packages/widgets/src/lib/components/PipelineIcon/PipelineIcon.svelte]: Sure, we also need one for Text to Audio. I'll see if I can make something for them in a follow-up PR.

**Pull request merged**
